container/brew.repo

```diff
@@ -1,6 +1,6 @@
 [brew]
 name=brew
-baseurl=http://download.eng.bos.redhat.com/brewroot/repos/brew-rhel-8/4285303/x86_64
+baseurl=http://download.eng.bos.redhat.com/brewroot/repos/brew-rhel-8-stage/latest/x86_64/
```

Comment:
This means it changes under us, right? I think that's ok in this case, because you said that it the others will disappear? Do you mind adding a comment about that?

Reply:
Yeah, it was the recommendation to use latest because it seems to get re-created. I can add a comment indeed.

Reply:
:heart: